src/randoop/operation/MethodCall.java

```diff
@@ -228,9 +235,8 @@ private void appendReceiverOrClassForStatics(String receiverString,
       Class<?> expectedType = getInputTypes().get(0);
       String canonicalName = expectedType.getCanonicalName();
       boolean mustCast = canonicalName != null
```

Comment:
I personally like to parenthesize compound boolean expressions, especially if they span multiple lines.  I think it makes the code easier to read.